pkg/arvo/app/launch.hoon

```diff
@@ -191,9 +191,14 @@
   ^-  (unit (unit cage))
   ?.  (team:title our.bowl src.bowl)  ~
   ?+  path  [~ ~]
-      [%x %tiles ~]       ``noun+!>([tiles tile-ordering])
-      [%x %first-time ~]  ``noun+!>(first-time)
-      [%x %keys ~]        ``noun+!>(~(key by tiles))
+    [%x %tiles ~]       ``noun+!>([tiles tile-ordering])
+    [%x %first-time ~]  ``noun+!>(first-time)
+    [%x %keys ~]        ``noun+!>(~(key by tiles))
+  ::
+      [%x %runtime-lag ~]
+    :^  ~  ~  %json
+    !>  ^-  json
+    b+.^(? //(scot %p our.bowl)//(scot %da now.bowl)/zen/lag)
```

Comment:
what's the `//` syntax doing here?

Reply:
Empty path segment. Scries into arvo use neither cares nor vane indicators, so the first segment is always the empty string. (Same for the desk.) An alternative way to write this would be `.^(? %$ /(scot ...`

Reply:
Yes, I see. All good then!